Rakefile

```diff
@@ -61,12 +61,8 @@ end
 desc "verify that commit messages match CONTRIBUTING.md requirements"
 task(:commits) do
   # This rake task looks at the summary from every commit from this branch not
-  # in the branch targeted for a PR. This is accomplished by using the
-  # TRAVIS_COMMIT_RANGE environment variable, which is present in travis CI and
-  # populated with the range of commits the PR contains. If not available, this
-  # falls back to `master..HEAD` as a next best bet as `master` is unlikely to
-  # ever be absent.
-  commit_range = ENV['TRAVIS_COMMIT_RANGE'].nil? ? 'master..HEAD' : ENV['TRAVIS_COMMIT_RANGE'].sub(/\.\.\./, '..')
+  # in the branch targeted for a PR.
+  commit_range = 'HEAD^..HEAD'
```

Comment:
github doesn't seem to like this https://github.com/puppetlabs/puppet/pull/8350/checks?check_run_id=1194014821#step:5:5

Reply:
updated the fetch depth for checkout action, should solve this.